homeassistant/components/tuya/sensor.py

```diff
@@ -1124,6 +1124,13 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             device_class=SensorDeviceClass.HUMIDITY,
             state_class=SensorStateClass.MEASUREMENT,
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.FAULT,
+            translation_key="fault",
+            device_class=SensorDeviceClass.ENUM,
+            state_class=SensorStateClass.MEASUREMENT,
```

Comment:
I don't think the device_class or the state_class makes sense.
And I would probably use entity_category `DIAGNOSTIC` for this.

Reply:
@epenet good point. Fixed.